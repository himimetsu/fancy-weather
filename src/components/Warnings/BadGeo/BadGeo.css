.bad-geo {
  width: 100%;
  padding: 0 0 30px 0;
  font-size: 35px;
  color: white;
  text-align: center;
}