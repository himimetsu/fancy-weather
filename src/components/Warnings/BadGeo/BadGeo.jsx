import React from 'react';
import './BadGeo.css';

const BadGeo = ({ lang }) => {
  return (
    <div className='bad-geo'>
      {lang === 'ru'
        ? ('Пожалуйста, предоставьте сайту информацию о вашей геолокации!')
        : lang === 'en'
          ? ('Please, provide the site with information about your location!')
          : ('Будь ласка, надайте сайту інформацію про вашу геолокації!')
      }
    </div>
  );
};

export default BadGeo;
