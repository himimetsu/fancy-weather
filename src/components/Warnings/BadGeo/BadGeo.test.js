import React from 'react';
import ReactDOM from 'react-dom';
import BadGeo from './BadGeo';
import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BadGeo />, div);
});

it('mathes snapshot', () => {
  const { asFragment } = render(<BadGeo />);
  expect(asFragment()).toMatchSnapshot();
})