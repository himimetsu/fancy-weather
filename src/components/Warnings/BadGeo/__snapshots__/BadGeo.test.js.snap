// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mathes snapshot 1`] = `
<DocumentFragment>
  <div
    class="bad-geo"
  >
    <span
      class="ru"
    >
      Пожалуйста, предоставьте сайту информацию о вашей геолокации!
    </span>
    <span
      class="en"
    >
      Please, provide the site with information about your location!
    </span>
    <span
      class="be"
    >
      Калі ласка, падайце сайту інфармацыю аб вашай геолокации!
    </span>
  </div>
</DocumentFragment>
`;
